microsetta_interface/templates/sitebase.jinja2

```diff
@@ -85,19 +84,19 @@
     <div>
         <br />
         <hr>
+          <p>We will periodically update this site with minor revisions; we
+          apologize if you encounter any temporary inconveniences.</p>
         {% if not disable_faq %}
         <p>
-            Questions?  Check our <a href="/view_faq" target="_blank">FAQ</a>!
-        </p>
+            Questions?  Check out our <a href="/view_faq" target="_blank">FAQ</a>!</br>
         {% endif %}
```

Comment:
Since the {% if ... %} {% endif %} surrounds the <p> tag, this looks wrong to me.  It means that if I render this page with disable_faq set to True, the <p> tag will not be rendered (but the </p> on line 98 will be).  We should always have matched tags within the {% %} control flow constructs.  

You can fix this in this case by moving the <p> up above the {% if not disable_faq %}